src/routes/datasets/[id].svelte

```diff
@@ -1,29 +1,91 @@
 <script context='module'>
+	import { writable } from 'svelte/store';
+	import {
+		aggregationsPerType,
+		determineESType,
+		buildAggregation
+	} from 'app/elasticsearch';
+	import {
+		request
+	} from 'app/net';
+
+	import JSONTree from 'svelte-json-tree'
```

Comment:
I'd put all npm imports together and the the local imports together separated with a blank line